### PR TITLE
Adding a video tutorial series for LLMs with Dotnet + Semantic Kernel

### PR DESCRIPTION
**Pull request by mahomedalid**
Btw, seems like this link is a dead link:

[Start your AI and .NET Adventure with #30DaysOfAzureAI](https://www.youtube.com/watch?v=567890)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Note: This list is currently hosted on GitHub by me, [Jordan Matthiesen](https:/
 
 - [Build your own Course Assistant with AI](https://youtu.be/BRaltelZt6U?si=uuUvRc_9jSW4L601) "Are you using Azure OpenAI Service to build Copilot applications? Do you understand the concept of Copilot Stack? In this session we will use Semantic Kernel to build your first Copilot application based on Copilot Stack." - Hosted by Luis Quintanilla (Microsoft) and Kinfey Lo (Microsoft)
 - [Start your AI and .NET Adventure with #30DaysOfAzureAI](https://www.youtube.com/watch?v=567890)
+- [Demystifying LLMS From Zero to Hero, with Azure OpenAI, DotNet, Semantic Kernel and OpenTelemetry](https://www.youtube.com/playlist?list=PLPK0YODPQ5naaUwDNFay0mlRwaV_AbrX6)
```
Comment (jmatthiesen) [inline README.md:56]: Could you please add a short description after this that tells the viewer what they'll build in this tutorial? Thanks!

**Comment by jmatthiesen** [comment]
Just one quick comment for you - thanks for the PR, and I'll also go ahead and remove that dead link that you mentioned.

File: README.md (hunk 1 above)
Comment (mahomedalid) [reply README.md:55]: Let me know if this is better.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Note: This list is currently hosted on GitHub by me, [Jordan Matthiesen](https:/
 ## Tutorials
 
 - [Build your own Course Assistant with AI](https://youtu.be/BRaltelZt6U?si=uuUvRc_9jSW4L601) "Are you using Azure OpenAI Service to build Copilot applications? Do you understand the concept of Copilot Stack? In this session we will use Semantic Kernel to build your first Copilot application based on Copilot Stack." - Hosted by Luis Quintanilla (Microsoft) and Kinfey Lo (Microsoft)
+- [A video series tutorial to create a production-ready LLM tool using Azure OpenAI, DotNet, Semantic Kernel and OpenTelemetry](https://www.youtube.com/playlist?list=PLPK0YODPQ5naaUwDNFay0mlRwaV_AbrX6)
```
Comment (jmatthiesen) [inline README.md:55]: Updated to match the pattern I'm using in the rest of the page (though I need to fix a few items still).

```suggestion
- [Semantic Kernel tutorial with dotnet: From Zero to Raccoon-Hero](https://www.youtube.com/playlist?list=PLPK0YODPQ5naaUwDNFay0mlRwaV_AbrX6) - A video series tutorial showing how to create a production-ready LLM tool using Azure OpenAI, .NET, Semantic Kernel and OpenTelemetry.
```

**Review by jmatthiesen — APPROVED** [review]

**Pull request merged**